packages/cmf/__tests__/cmfModule.merge.test.js

```diff
@@ -241,14 +238,14 @@ describe('mergeModule', () => {
 
 		// when
 		const { RootComponent } = mergeModules(module1, module2, module3);
-		const wrapper = mount(<RootComponent />);
+		const { container } = render(<RootComponent />);
 
 		// then
-		const mod1 = wrapper.find('#mod1');
+		const mod1 = container.querySelectorAll('#mod1');
```

Comment:
The goal of React Testing Library is to test the rendering like a user does, AFAIK.
Can we rely on the content of the #mod1,2,3 instead? 🤔